docs/configuration/disk-trim.md

```diff
@@ -26,6 +26,7 @@ Configuration
 
 In all versions of Qubes, you may want to set up a periodic job in `dom0` to trim the disk.
 This can be done with either systemd (weekly only) or cron (daily or weekly).
+You should add this service or cron to **each VM** if you want the trim function to work in.
```

Comment:
How does this sound?
You should also add this service or cro jobn to **each VM** you want the trim function to work in.

Reply:
I agree.
I'm not fluent with the github review mechanism, should I do a new commit or just accept your review somewhere?

Reply:
You should be able to revise your earlier commit and keep this same pull, but it's fine if you just want to cancel this and resubmit as new.